src/testing/flow_solver_cases/naca0012.h

```diff
@@ -0,0 +1,51 @@
+#ifndef __NACA0012__
+#define __NACA0012__
+
+// for FlowSolver class:
+#include "physics/initial_conditions/initial_condition.h"
+#include "dg/dg.h"
+#include "physics/physics.h"
+#include "parameters/all_parameters.h"
+#include <deal.II/base/table_handler.h>
+#include <deal.II/distributed/shared_tria.h>
+#include <deal.II/distributed/tria.h>
+#include "flow_solver_case_base.h"
+
+namespace PHiLiP {
+namespace Tests {
+
+#if PHILIP_DIM==1
+    using Triangulation = dealii::Triangulation<PHILIP_DIM>;
+#else
+    using Triangulation = dealii::parallel::distributed::Triangulation<PHILIP_DIM>;
+#endif
```

Comment:
Move those using inside the class. Otherwise those using become "global" to the PHiLiP::Tests namespace when this file is included.

Reply:
Fixed in most recent commit.